solutions/observability/infra-and-hosts/configure-settings.md

```diff
@@ -25,7 +25,7 @@ From the main menu, go to **Infrastructure** → **Infrastructure inventory** or
 Click **Apply** to save your changes.
 
 ::::{note}
-The patterns used to match log sources are configured in the Logs app. The default setting is `logs-*,filebeat-*,kibana_sample_data_logs*`. To change the default, refer to [Configure data sources](../../../solutions/observability/logs/configure-data-sources.md).
+The patterns used to match log sources are configured in the Logs app. The default setting is `logs-*,filebeat-*,kibana_sample_data_logs*`. To change the default, go to **Log sources** at **Stack Management** → **Advanced settings** under **Observability**.
```

Comment:
The patterns are only configured through the Kibana advanced settings, we could remove the reference to the Logs app. The current default seems to be `logs-*-*, logs-*, filebeat-*, kibana_sample_data_logs*`.